Firmware/ESP32/Arduino_platform/lib/DEV/dev_imu.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file dev_imu.h
+ * @author Dong Jae (Alex) Park
+ * @date 18 Feb 2021
+ * @brief Device IMU
+ * 
+ * This document will contains device configure content
+ */
+
+
+#ifndef DEV_IMU_H
+#define DEV_IMU_H
+# ifdef __cplusplus
+extern "C"{
+# endif
+
+// External Lib
+#include <stdbool.h>
+
+typedef enum {
+    IMU_AXIS_ACC_X,
+    IMU_AXIS_ACC_Y,
+    IMU_AXIS_ACC_Z,
+    IMU_AXIS_GYR_X,
+    IMU_AXIS_GYR_Y,
+    IMU_AXIS_GYR_Z,
+    IMU_AXIS_MAG_X,
+    IMU_AXIS_MAG_Y,
+    IMU_AXIS_MAG_Z,
+    IMU_AXIS_IMU_COUNT,
+    IMU_AXIS_IMU_UNDEFINED
+} IMU_AXIS_E;
+
+void dev_imu_init(void);
+bool dev_imu_get_values(float* data_ptr);
```

Comment:
add function comment header:
+ expectation: invoke this function will request imu data via SPI. (request based)